core/dbt/task/printer.py

```diff
@@ -128,11 +128,11 @@ def print_test_result_line(
         color = ui.green
         logger_fn = logger.info
     elif result.status == TestStatus.Warn:
-        info = 'WARN {}'.format(result.message)
+        info = 'WARN'
```

Comment:
I think parity with current would be something like `'WARN {}'.format(result.failures)`. Do you have a reason to prefer a clean `WARN` / `FAIL` info line? If so, I'm open to it!

Reply:
Adding a new field to run results for one type of task is going to turn into an entire refactor of how we handle task results. I'm shelving this for this issue.

Reply:
That's a good enough reason for me